client/src/actions/repoActions.js

```diff
@@ -1,130 +1,186 @@
 import Types from './actionTypes';
 import { createAction } from 'redux-actions';
+import Toastr from 'toastr';
 import * as api from '../utils/api';
 import apiPaths from '../utils/apiPaths';
-import Toastr from 'toastr';
+import store from '../store/configureStore';
 
-export function fetchRepoFailure(error) {
-  return createAction(Types.RECEIVE_REPO_FAILURE)(error);
+export function fetchPersonalRepos(name, token) {
+    return dispatch => {
+        dispatch(fetchPersonalReposRequest());
+        return api.githubFetch(`/users/${name}/repos?`, token)
+            .then(personalRepos => {
+                return api.callEndpoint(`${apiPaths.USER_EP}/${personalRepos[0].owner.id}${apiPaths.REPOS_EP}`)
+                    .then(watchedRepos => {
+                        const data = [];
+                        personalRepos.map(pRepo => {
+                            let found = false;
+                            watchedRepos.map(wRepo => {
```

Comment:
Since you're not using the data returned by the map function, consider using forEach